functions/pipeline/shared/db_access/db_access_v2.py

```diff
@@ -27,14 +25,18 @@ def __init__(self, image_name, image_location, height, width):
         self.height = height
         self.width = width
 
+
 class ImageTag(object):
-    def __init__(self, image_id, x_min, x_max, y_min, y_max, classification_names):
-            self.image_id = image_id
-            self.x_min = x_min
-            self.x_max = x_max
-            self.y_min = y_min
-            self.y_max = y_max
-            self.classification_names = classification_names
+    def __init__(self, image_id, x_min, x_max, y_min, y_max, classification_names, image_height, image_width):
+        self.image_id = image_id
+        self.x_min = x_min
+        self.x_max = x_max
+        self.y_min = y_min
+        self.y_max = y_max
+        self.classification_names = classification_names
+        self.image_height = image_height
```

Comment:
@mtarng so this doesn't reflect the db table now? It seems like we have overloaded this class to fit multiple needs. Is that the case?